src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml

```diff
@@ -6,6 +6,10 @@
 					xmlns:ios="http://nventive.com/ios"
 					mc:Ignorable="xamarin android ios">
 
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="MaterialOverride.xaml" />
```

Comment:
I don't think we want to merge MaterialOverride.xaml more than once.
@MatFillion, @kazo0

Reply:
Correct that probably doesn't work.